org.mwc.debrief.lite/src/main/java/org/mwc/debrief/lite/gui/custom/JXCollapsiblePaneWithTitle.java

```diff
@@ -184,7 +188,15 @@ public void mouseDragged(final MouseEvent event)
             }
 
             bounds = getBounds();
-            bounds.height = (bounds.height - oldHeight) + newDimension;
+            final int maxheight = DebriefLiteApp.getInstance().getApplicationFrame().getHeight();
+            //165 is the height of ribbon
```

Comment:
@ayeshare - can't we determine the actual current height of the ribbon?

That would be a longer term fix

Reply:
Replaced the fixed height with height of ribbon.